solo_chaos/cult_leader

```diff
@@ -5,6 +5,7 @@ Cult Leader- SOLO -
 - The added person is now on the cult's team and has the same win goal as the cult leader (The cult leader is now their new leader). The converted people still have access to the old team chats and buildings.
 - The added person will not lose their abilities and will still have access to their old team chat.
 - If a king is converted, they still act as a king to their own team, but their allegiance lies with the Cult Leader. If a solo was supposed to be converted, the conversion will fail, and the curse is removed from the game.
-- The cult leader can cast a curse once a phase. During the night, the curse is cast after all the kills. They can only curse alive people. If the person who was supposed to be cursed dies, the curse will be failed and the cult leader is informed.
+- The cult leader can cast a curse once a phase. They can only curse alive people. If the cult leader attempts to curse someone who dies that night or by tribute (if it's day), the curse will fail and the curse is still consumed.
```

Comment:
Why is the curse consumed? I feel like that's a huge disadvantage

Reply:
incentive to pay attention to the game and strategize, in order to choose correctly

Reply:
Alright then, no need to remove it.